docs/registry/entities/workflow.md

```diff
@@ -0,0 +1,181 @@
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/entity_namespace.md.j2 -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+# Workflow
+
+## Workflow
+
+**Status:** ![Development](https://img.shields.io/badge/-development-blue)
+
+**type:** `workflow`
+
+**Description:** 
+
+**Identifying Attributes:**
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`workflow.id`](/docs/registry/attributes/workflow.md) | string | Unique ID for this workflow which can be used to correlate multiple executions of this workflow. | `5c6e9a4b-69ab-499f-b4a7-4cf5c8720d66` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
+
+**Descriptive Attributes:**
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`workflow.name`](/docs/registry/attributes/workflow.md) | string | Name of the workflow | `Build and deploy application` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
+| [`workflow.type`](/docs/registry/attributes/workflow.md) | string | Type of the workflow | `DSL` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
+| [`workflow.url.full`](/docs/registry/attributes/workflow.md) | string | The [URL](https://wikipedia.org/wiki/URL) of the workflow, providing the complete address in order to locate and identify the workflow. | `https://github.com/open-telemetry/semantic-conventions/actions/runs/9753949763?pr=1075` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
+| [`workflow.version`](/docs/registry/attributes/workflow.md) | string | Version of the workflow | `2.1` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
+
+## Workflow Execution
+
+**Status:** ![Development](https://img.shields.io/badge/-development-blue)
+
+**type:** `workflow.execution`
+
+**Description:** 
+
+**Identifying Attributes:**
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`workflow.execution.id`](/docs/registry/attributes/workflow.md) | string | Unique ID for this execution of the workflow and will be shared across all the task runs. | `5c6e9a4b-69ab-499f-b4a7-4cf5c8720d66` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
Considering tracing, would the `workflow.execution.id` (or `workflow.run.id` as suggested in another comment), have the same value as the trace id?